applications/DEM_application/DEM_application.cpp

```diff
@@ -70,7 +69,18 @@ KRATOS_CREATE_VARIABLE(std::string, EXCENTRICITY_PROBABILITY_DISTRIBUTION)
 
 // OPTIONS AND FLAGS
 KRATOS_CREATE_VARIABLE(int, FORCE_INTEGRATION_GROUP)
-KRATOS_CREATE_VARIABLE(int, TABLE_NUMBER)
+KRATOS_CREATE_VARIABLE(int, TABLE_NUMBER_VELOCITY_X)
```

Comment:
Just a aesthetics comment. Would it be possible to create 3 variable vectors of int instead of 9 ints?

Reply:
I have been thinking a lot about the way to organize this. As the tables can only have two columns (TIME VARIABLE) this was the way I finally did it. Moreover, in some cases, we would only define the velocity/force in one direction and let the others free. Probably there is a more elegant way to define this but I did not find it.

Reply:
Finally I figured out how to do it.